events/wc_druidic_events.txt

```diff
@@ -209,7 +209,7 @@ narrative_event = {
 	picture = GFX_evt_dalaran
 	border = GFX_event_narrative_frame_war
 	
-	is_triggered_only = yes
+	fire_only_once = yes
```

Comment:
Never seen it. Wrong syntax?

Reply:
This is HOI thing.
It's don't work in ck2, tho.

Reply:
Unfortunate. Hopefully it will get added in a later patch.